media/CircleEditor/CircleEditor.md

```diff
@@ -0,0 +1,13 @@
+## Circle Editor
+
+This extension will add circle editor that enables modification in `*.circle` files and provides JSON formatted data for the circle file.
+
+### Circle Editor UI Description
```

Comment:
Let's only leave simple one sentence and then write details after overall features are all merged :)
Please remove this section